test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+import traceback
+import unittest
+
+from configparser import ConfigParser
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+
+
+def assert_exception_correct(got: Exception, expected: Exception):
+    """
+    Compare raised exception with expected exception.
+
+    Args:
+        got (Exception): Exception received
+        expected (Exception): Exception expected
+    """
+    err = "".join(traceback.TracebackException.from_exception(got).format())
+    assert got.args == expected.args, err
+    assert type(got) == type(expected)
+
+
+class ImplTest(unittest.TestCase):
+    def setUp(self):
+        config_file = os.environ["KB_DEPLOYMENT_CONFIG"]
```

Comment:
No need to read the config file for a unit test (and in a unit test you should avoid reading external files anyway), just make a config map

Reply:
updated!

Reply:
:+1: